.github/workflows/nodejs.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [12.x]
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
I think checking lint one time is enough. 

Tough we can try enable node 16 for tests -

https://github.com/webpack/webpack-dev-server/blob/7e8692286c1cdb9b1e1483c3cf425cec849d9783/.github/workflows/nodejs.yml#L60

Reply:
Yes, we should enable it for tests

Reply:
oh yep intention was to enable for test, updating